src/other/utils.jl

```diff
@@ -2,10 +2,13 @@ import Base: isidentifier, is_id_start_char, is_id_char
 
 const RESERVED_WORDS = Set(["begin", "while", "if", "for", "try",
     "return", "break", "continue", "function", "macro", "quote", "let",
-    "local", "global", "const", "abstract", "typealias", "type", "bitstype",
+    "local", "global", "const", "type",
     "immutable", "do", "module", "baremodule", "using", "import", "struct",
     "export", "importall", "end", "else", "elseif", "catch", "finally"])
 
+if VERSION < v"0.7.0-DEV.3220"
```

Comment:
Didn't this happen in 0.6-dev?

Reply:
No, or the tests would fail on 0.6. This commit appears to be the one which removed these deprecated identifiers.